app/controllers/problems_searcher.rb

```diff
@@ -25,6 +25,10 @@ module ProblemsSearcher
       Array(Problem.find(err_ids))
     end
 
+    expose(:selected_problems_ids) do
+      selected_problems.map(&:id).map(&:to_s)
```

Comment:
This is needed because if you try passing an Array of BSON::ObjectId objects you get 
ActiveJob::SerializationError: Unsupported argument type: BSON::ObjectId

ActiveJob::SerializationError:
       Unsupported argument type: BSON::ObjectId

with this stacktrace
     .../gems/activejob-4.2.5.2/lib/active_job/arguments.rb:69:in `serialize_argument'
     .../gems/activejob-4.2.5.2/lib/active_job/arguments.rb:58:in `block in serialize_argument'
     .../gems/activejob-4.2.5.2/lib/active_job/arguments.rb:58:in `map'

So the simplest way is calling to_s, which gave you a String representation of that ObjectId

and build a BSON::ObjectId with the method from_string